.github/workflows/test.yml

```diff
@@ -51,8 +51,14 @@ jobs:
       - name: Fetch cache
         uses: Swatinem/rust-cache@a95ba195448af2da9b00fb742d14ffaaf3c21f43 # v2.7.0
 
+      - name: Test
+        run: cargo test --workspace --release --locked -q --features=runtime-metrics,try-runtime
```

Comment:
Well, the problem with `runtime-benchmarks` is that it can be used for `pallet_xyz::Config` configuration,
e.g.:  https://github.com/polkadot-fellows/runtimes/blob/main/relay/kusama/src/lib.rs#L1266-L1270
```
        #[cfg(not(feature = "runtime-benchmarks"))]
	type MessageProcessor = MessageProcessor;
	#[cfg(feature = "runtime-benchmarks")]
	type MessageProcessor =
		pallet_message_queue::mock_helpers::NoopMessageProcessor<AggregateMessageOrigin>;
```
which can lead to misleading unit test results `MessageProcessor` vs `NoopMessageProcessor`.

When it comes to `runtime-metrics` or `try-runtime`, I have never seen them being used for `pallet_xyz::Config` configuration.

I'm not sure if running all combinations is worth it. Perhaps this would be enough?
```
cargo test --workspace --release --locked -q 
cargo test --workspace --release --locked -q --features=runtime-metrics,try-runtime
cargo test --workspace --release --locked -q --features=runtime-metrics,try-runtime,runtime-benchmarks
```